### ISSUE TITLE
WCAG 4.1.2 test 9: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#94]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7396819/test-document.pdf) fails **WCAG 4.1.2 test 9** ('_Missing table element_'). The error is given for the cells in the first column of the table on page 2. This is a correctly tagged table, so these appear to be false positives.

Note that the algorithm also gives this error on the aligned texts on pages 1 and 6. These are not false positives, but it's clear these texts were not intended as tables because there are only two columns and no text that appears to be intended as header cells.

Perhaps the algorithm should not trigger WCAG 4.1.2 test 9 unless:

- There is clear evidence of text intended as table head (e.g. the first line of aligned text is set in boldface), **and**
- there appear to be more than 2 columns

What do you think?

**Comment by bdoubrov** [comment]
> Note that the algorithm also gives this error on the aligned texts on pages 1 and 6. These are not false positives, but it's clear these texts were not intended as tables because there are only two columns and no text that appears to be intended as header cells.

I'd say that the first column in both cases can be viewed as table heading. Alternatively, may be considered as a definition list. But certainly not as a plain paragraph, as there is a certain additional semantics here.

**Comment by jzuidweg** [comment]
Although **WCAG 4.1.2 test 9** ('_Missing table element_') keeps giving false positives on other test documents, it now behaves correctly on the test file provided with this issue. I'm therefore closing the issue.
